src/components/spreadsheet/table-wrapper.js

```diff
@@ -202,6 +206,36 @@ const TableWrapper = (props) => {
         );
     }, [props.disabled, selectedColumnsNames, tabIndex]);
 
+    const equipementFiltersSelectorKeys = useMemo(() => {
+        let filtersSelectorKeys = {};
+        TABLES_DEFINITION_INDEXES.get(tabIndex).columns.forEach((column) => {
+            filtersSelectorKeys[column?.field] = column?.field;
+        });
+        return filtersSelectorKeys;
+    }, [tabIndex]);
+
+    const defaultSortColKey = useMemo(() => {
+        const defaultSortCol = columnData.find(
+            (column) => column.isDefaultSort
+        );
+        return defaultSortCol?.field;
+    }, [columnData]);
+
+    const { onSortChanged, sortConfig, initSort } = useAgGridSortImpl(gridRef, {
+        colKey: defaultSortColKey,
+        sortWay: SORT_WAYS.asc,
+    });
+
+    // This is a dummy callback to avoid the warning of missing dependencies and infinite loop
+    const updateFilterCallback = useCallback(() => {}, []);
+
+    const { updateFilter, filterSelector, initFilters } =
+        useAggridRowFilterImpl(
+            gridRef,
+            equipementFiltersSelectorKeys,
+            updateFilterCallback
+        );
```

Comment:
I think it would be cleaner to change "useAggridRowFilter" definition and make "updateFilterCallback" not mandatory

you could then remove "updateFilterCallback" from "useAggridRowFilterImpl" props

Reply:
It's already not mandatory, i deleted the dummy function, we don't need it anymore